save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -113,6 +139,68 @@ class WarnPlugin(testConfig: TestConfig, testFiles: List<String> = emptyList())
         )
     }
 
+    @Suppress(
+        "TOO_LONG_FUNCTION",
+        "SAY_NO_TO_VAR")
+    private fun handleTestFile(
```

Comment:
Functions for single file and for batch should be unified, now it's a lot of copy-paste